packages/yourloops/components/statistics/patient-statistics.tsx

```diff
@@ -90,6 +100,17 @@ export const PatientStatistics: FunctionComponent<PropsWithChildren<PatientStati
       }
 
       {children}
+
+      <CoefficientOfVariation coefficientOfVariation={coefficientOfVariation} bgSource={bgSource} />
+      <Divider sx={{ marginBlock: theme.spacing(1), backgroundColor: theme.palette.grey[600] }} />
+
+      {isTrendsPage && !cbgSelected &&
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏Just a thought (nothing to do), but wouldn't it be simpler to have a stat component for Trends, one for Daily and one for dashboard? @fatb38

Reply:
Nice idea @TimCanu, until there it wasn't usefull and also we did'nt know exactly which components for which page. But now I think it will be better to have a component for each 😉